README.rst

```diff
@@ -1,11 +1,59 @@
 KEG - Image Compositon Tool
 ===========================
 
-.. |Build Status| image:: https://travis-ci.com/SUSE-Enceladus/keg.svg?branch=main
-   :target: https://travis-ci.com/SUSE-Enceladus/keg
-
-|Build Status|
+.. ![CILint](https://github.com/SUSE-Enceladus/keg/workflows/CILint/badge.svg?branch=main)
 
 keg is a command line tool that creates a
 [kiwi](https://github.com/OSInside/kiwi) image description based on
 description snippets in a given GIT repository.
+
+
+## Contents
+
+  * [Contributing](#contributing)
+
+## Contributing
+
+keg is written in Python, it uses [tox](https://tox.readthedocs.io/en/latest/) to setup a development environment
+for the desired Python version. Make sure the Python development headers
+are installed (e.g. `python36-devel`)
+
+The following procedure describes how to create such an environment:
+
+1.  Let tox create the virtual environment(s):
+
+    ```
+    $ tox
```

Comment:
could be ```tox -e devel```  just tox does everything also the docs and the checks and all of it. I think it would be nice if we mention the tox targets that exists. Thoughts

Reply:
Done